Generate.py

```diff
@@ -514,7 +516,11 @@ def handle_option(ret: argparse.Namespace, game_weights: dict, option_key: str,
             if hasattr(player_option, "verify"):
                 player_option.verify(AutoWorldRegister.world_types[ret.game])
     else:
-        setattr(ret, option_key, option(option.default))
+        if option.default == "random":
```

Comment:
this can be changed to `elif`. Wondering why this change is required though.

Reply:
There was recently a PR to allow random as the default value. If random is the default value for an option and said option is not present in a submitted yaml file it will assign "random" as the value instead of making a random choice. By calling the options `from_any` what happens can still be controlled by the option.